### ISSUE TITLE
OmniSharp isn't picking up FxCop Analyzer settings in top-level .editorconfig

### ISSUE THREAD

**Issue by alexrp** [OmniSharp/omnisharp-roslyn#1764]
(VS Code 1.44.0 / OmniSharp 1.34.15)

In my top-level `.editorconfig`, I have:

```editorconfig
[*.cs]
dotnet_diagnostic.ca2007.severity = none
```

And in my ´omnisharp.json`:

```json
{
  "FormattingOptions": {
    "EnableEditorConfigSupport": true
  },
  "RoslynExtensionsOptions": {
    "EnableAnalyzersSupport": true
  }
}
```

When I run `dotnet build`, CA2007 is indeed disabled as expected. However, in VS Code, I still see CA2007 warnings. I'm not sure whether this is a bug or I'm just doing something wrong.

**Comment by filipw** [comment]
There is no support for that at the moment.
It's already tracked in a separate issue though https://github.com/OmniSharp/omnisharp-roslyn/issues/1657#issuecomment-611880861

**Comment by alexrp** [comment]
Is there at least a way to disable analyzers pulled in from NuGet?

As it stands, `EnableAnalyzersSupport=false` seems to disable absolutely everything, including `IDE*` analyzers. But setting `EnableAnalyzersSupport=true` is near-unusable due to all the false positives that show up due to OmniSharp not picking up `.editorconfig`.

**Comment by filipw** [comment]
yes, you can create a ruleset file
